modyn/supervisor/internal/triggers/drift/drift_detector.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC
+
+import numpy as np
+import pandas as pd
+import torch
+from modyn.config.schema.pipeline.trigger.drift.config import DriftMetric
+from modyn.config.schema.pipeline.trigger.drift.result import MetricResult
+
+
+class DriftDetector(ABC):
```

Comment:
Is this for your custom MMD implementation?

Reply:
No, it's just the interface that abstracts from alibi-detect and evidently. I removed my custom implementation for now as we don't need it.

Reply:
ah sorry, i did not realize it's the abstract base class